guides/create-graphql-query.md

```diff
@@ -0,0 +1,241 @@
+## At a glance
+
+You're working on a new implementation for Leader of the Pack, an outdoor equipment retailer that we just took 
```

Comment:
We're working on a new implementation for Leader of the Pack, the outdoor equipment retailer that we just [built an API plugin](https://mailchimp.com/developer/open-commerce/guides/build-api-plugin/) for. The store stocks kayaks, but can't ship them to every location because of the logistical challenges of shipping something that large. We need a way to show if a particular product can be shipped to a particular ZIP code.

In this guide, we'll walk you through the process of creating a GraphQL query and having it recognized by the system.